File: namsor/python/src/JapaneseNameKanjiCandidates1Example.py
import json
from datetime import date, datetime
from pprint import pprint

from namsor_client import ApiClient, ApiException, Configuration, api, models

configuration = Configuration(
    api_key={"api_key": "YOUR_API_KEY"},
)

with ApiClient(configuration) as api_client:
    try:
        response = api.JapaneseApi(api_client).japanese_name_kanji_candidates_1(
            japanese_surname_latin="Sanae",
            japanese_given_name_latin="Yamamoto",
        )

        pprint(response)
    except ApiException as e:
        print("Exception when calling JapaneseApi#japanese_name_kanji_candidates_1: %s\n" % e)

File: namsor/ruby/src/JapaneseNameKanjiCandidates1Example.rb
require "json"
require "namsor_client"

NamsorClient.configure do |config|
    config.api_key["X-API-KEY"] = "YOUR_API_KEY"
end

begin
    response = NamsorClient::JapaneseApi.new.japanese_name_kanji_candidates_1(
        "Sanae", # japanese_surname_latin
        "Yamamoto", # japanese_given_name_latin
    )

    p response
rescue NamsorClient::ApiError => e
    puts "Exception when calling JapaneseApi#japanese_name_kanji_candidates_1: #{e}"
end
